.github/workflows/dalgo-ci.yml

```
@@ -81,15 +81,16 @@ jobs:
           coverage xml
           coverage report --fail-under=70
 
+      - name: Upload coverage reports to codecov
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.4)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)